static/src/javascripts/projects/commercial/modules/build-page-targeting.js

```diff
@@ -123,8 +123,11 @@ const formatAppNexusTargeting = (obj: Object): string =>
             })
     ).join(',');
 
-const buildPageTargeting = once((): Object => {
+const buildPageTargeting = once((adFree: ?boolean): Object => {
     const page: Object = config.page;
+    const adFreeTargeting: Object = adFree
+        ? Object.assign({ af: 't' })
+        : Object.assign({});
```

Comment:
By that token, can't we use just `{af: 't'}` intead of the `.assign` too? I used the same technique that had been applied elsewhere.